test/test_tree.rb

```diff
@@ -1413,38 +1410,24 @@ def test_old_camelCase_method_names
                               isOnlyChild? nextSibling previousSibling nodeHeight nodeDepth
                               removeFromParent! removeAll! freezeTree! }
 
-      require 'structured_warnings'
-
-      DeprecatedMethodWarning.disable do
-        assert(@root.isRoot?)   # Test if the original method is really called
```

Comment:
Previous version of a unit test checked that original method is really called when a camel-cased method is run. So a new version should arguably continue to check it.